srcs/Webserv/Request.hpp

```diff
@@ -0,0 +1,217 @@
+#ifndef SRCS_WEBSERV_REQUEST_HPP_
+#define SRCS_WEBSERV_REQUEST_HPP_
+
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "./Response.hpp"
+#include "./Socket.hpp"
+#include "./utils.hpp"
```

Comment:
```suggestion
#include "Response.hpp"
#include "Socket.hpp"
#include "utils.hpp"
```
ちゃんとヘッダーのパスが通せていれば相対パス形式で指定しなくて良いと思います。

Reply:
これディレクトリを指定しないとlinterに怒られるんですよね

Reply:
そうなんですねー。